src/test/py/bazel/bazel_workspace_test.py

```diff
@@ -0,0 +1,58 @@
+# pylint: disable=g-bad-file-header
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from src.test.py.bazel import test_base
+
+
+class BazelWorkspaceTest(test_base.TestBase):
+
+  def testWorkspaceDotBazelFileInMainRepo(self):
+    self.ScratchFile("WORKSPACE.bazel")
+    self.ScratchFile("BUILD", [
+      "py_binary(",
+      "  name = 'bin',",
+      "  srcs = ['bin.py'],",
+      ")",
+    ])
+    self.ScratchFile("bin.py")
+    exit_code, _, stderr = self.RunBazel(['build', '//:bin'])
```

Comment:
Please also add:
- a negative test: delete or rename WORKSPACE.bazel and assert the build no longer works.
- a test for renaming to "WORKSPACE"

opt: No need for the py_binary, just an empty BUILD file. If `bazel build //:BUILD` works, then the workspace is valid. Also in tests below.

Reply:
Thanks, I added code for testing the build after removing the `WORKSPACE.bazel` file. One without `WORKSPACE`, the other case has `WORKSPACE`.

Reply:
I'll leave the python targets as it is because I want to test the build actually works and works with external dependencies.